composer/datasets/ffcv_utils.py

```diff
@@ -109,3 +111,53 @@ def write_ffcv_dataset(dataset: Optional[Dataset] = None,
         # We call this internal API instead of writer.from_webdataset because with writer.from_webdataset
         # FFCV downloads the whole dataset twice.
         writer._write_common(total_len, todos, ffcv.writer.worker_job_webdataset, (pipeline,))
+
+
+if ffcv_installed:
```

Comment:
Instead of conditioning on the FFCV install here, is it possible to condition in the individual methods, and then raise an exception if one attempts to use it but FFCV is not installed?